app/src/main/java/org/greenstand/android/TreeTracker/orgpicker/OrgPickerScreen.kt

```diff
@@ -53,17 +69,11 @@ fun OrgPickerScreen(viewModel: OrgPickerViewModel = viewModel(factory = LocalVie
             )
         },
     ) {
-        LazyColumn(
+        LazyVerticalGrid(
+            cells = GridCells.Fixed(2),
             modifier = Modifier.fillMaxSize(),
-            horizontalAlignment = Alignment.CenterHorizontally,
-            verticalArrangement = Arrangement.Center
+            horizontalArrangement = Arrangement.SpaceEvenly
```

Comment:
Can we please center align the list items vertically?

Reply:
We could change the design to this also. Should it still be grid or just single Columns?

Reply:
A grid is still good, but since most cases this will be only 2 items, it will look nicer if they are centered.